docs/src/test/java/docs/home/persistence/CqrsIntegrationTest.java

```diff
@@ -83,7 +83,7 @@ public static void setup() throws Exception {
 
   @AfterClass
   public static void teardown() {
-    application.getWrappedApplication().stop();
```

Comment:
Changing this one too since `getWrappedApplication` will be deprecated in favor of `asScala`.